flocker/node/script.py

```diff
@@ -6,25 +6,25 @@
 """
 
 from socket import socket
-from os import getpid
 
 import yaml
 
 from jsonschema import FormatChecker, Draft4Validator
 
-from pyrsistent import PRecord, field
+from pyrsistent import PRecord, field, PMap, pmap, pvector
 
 from zope.interface import implementer
 
 from twisted.python.filepath import FilePath
 from twisted.python.usage import Options
 from twisted.internet.ssl import Certificate
-
-from ..volume.service import (
-    ICommandLineVolumeScript, VolumeScript,
-)
+from twisted.internet import reactor
+from twisted.python.constants import Names, NamedConstant
 
 from ..volume.script import flocker_volume_options
+from ..volume.filesystems import zfs
```

Comment:
Will there be any problems importing `zfs` here if ZFS package is not installed?